### ISSUE TITLE
Missing notes in the Permanent Notes Folder

### ISSUE THREAD

**Issue by KelSolaar** [groepl/Obsidian-Zettelkasten-Starter-Kit#8]
Hello,

It would seem like the `Permanent Notes.md` file is missing from this folder: https://github.com/groepl/Obsidian-Zettelkasten-Starter-Kit/tree/main/Starter-Kit/3_Permanent%20Notes

![image](https://github.com/groepl/Obsidian-Zettelkasten-Starter-Kit/assets/99779/13b19fb4-cd13-470b-af26-b449ff2f380c)

Likewise, a few notes, e.g. are linked from the `Literature Notes` but are missing, maybe purposely:

![image](https://github.com/groepl/Obsidian-Zettelkasten-Starter-Kit/assets/99779/239e405b-30d3-484a-b4fa-6db0db04c679)

Cheers,

Thomas

**Comment by groepl** [comment]
You are right, the file is missing. To understand the background: The Zettelkasten-Starter-Kit is a subset of my active Zettelkasten. It is not a separate demo vault. This means that all links to further files remain open. 

Your options:
- Delete the open link.
- Create a new file to replace the missing file from the open link.

Your benefit:
- You will always get the latest files from my active vault with the next update.
